Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
This sounds a bit odd. The reference.ts file should not contain any Windows endings (`\r\n`). Would this work if you set VS to use UNIX line endings. [how to](http://stackoverflow.com/questions/3802406/configure-visual-studio-with-unix-end-of-lines)